README.md

```diff
@@ -143,14 +143,17 @@ fastify.ready(err => {
       path: './examples/example-static-specification.yaml',
       postProcessor: function(swaggerObject) {
         return swaggerObject
-      }
-    }
+      },
+      baseDir: '/path/to/external/spec/files/location',
+    },
   }
   ```
   Example of the `fastify-swagger` usage in the `static` mode is available [here](examples/static-file.js).
 
   `specification.postProcessor` parameter is optional. It allows you to change your swagger object on the fly (for example - based on the environment). It accepts `swaggerObject` - basically a javascript object which was parsed from your `yaml` or `json` file and should return a swagger object.
 
+  `specification.baseDir` allows to specify directory where all spec files which are included into the main one using `$ref` live. 
+  By default it's the directory where main spec file is located. Should be absolute path without trailing slash.
```

Comment:
allows specifying the directory where all spec files that are included in the main one using `$ref` will be located .  By default, this is the directory where the main spec file is located. Provided value should be an absolute path **without** trailing slash.

Reply:
Done